app/assets/javascripts/hyrax/save_work/visibility_component.es6

```diff
@@ -14,6 +14,7 @@ export default class VisibilityComponent {
     this.form.on('submit', () => { this.enableAllOptions() })
     this.showForm()
     this.limitByAdminSet()
+    element.data('visibility_component', this)
```

Comment:
@j-dornbusch This is setting `data-visibility_component="... this "` on the passed in element?  Do you need all the data from `this`?   Does the value need to be serialized?   I'm not sure how you're using here, just wondering if the value could be lighter.